src/scalability/introduction.md

```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+In the database field, scalability is the term we use to describe the capability of a system to handle a growing amount of work. For example, even if a system is working reliably and fast today, it doesn't mean it will necessarily work well in the future. One common reason for degradation is the increased load which exceeds the system can process. Nowadays, the speed of data growth far beyond our imagination, so scalability is a substantial part for the design of a database.
```

Comment:
```suggestion
In the database field, scalability is the term we use to describe the capability of a system to handle a growing amount of work. Even if a system is working reliably and fast today, it doesn't mean it will necessarily work well in the future. One common reason for degradation is the increased load which exceeds the system can process. In modern systems, the amount of data we handle can far outgrow our original expectations, so scalability is a critical consideration for the design of a database.
```